### PR TITLE
Timeslider

### PR DESCRIPTION
**Pull request by lindsaycarr**
~The one issue I am having is putting the slider at the correct location on the map. I ended up hard coding a value in because `chart_height - (chart_height * 0.15)` to bring it 15% from the bottom edge did not work. It seems that the map has decided to have different dimensions than what I setup for the SVG. Hoping to discuss this at the meeting later today.~

The slider is discrete (snaps to the years provided). At this point, nothing happens when the slider is changed, but I think we could easily return `round_yr` from the `dragged` function to subset data based on the slider actions.

slider

### CODE REVIEW

File: js/build_map.js [hunk 1]
```diff
@@ -33,5 +36,6 @@ function create_map() {
 	var state_data = arguments[1];
   
   add_states(map, state_data);
+  add_timeslider(map, years, chart_width, chart_width);
```
Comment (lindsaycarr) [inline js/build_map.js]: wait ... I think I just solved my slider placement issue. I was passing chart_width in twice so it was using the wrong value for chart_height inside this function. Fix on its way!

**Comment by lindsaycarr** [comment]
Chart_height issue fixed. Now located here:
image

File: js/timeslider.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ function add_timeslider(map, years, chart_width, chart_height) {
   
   var slider = map.append("g")
     .attr("id", "slider")
-    .attr("transform", "translate(0," + 575 + ")");
+    .attr("transform", "translate(0," + (chart_height - (chart_height * 0.05)) + ")");
```
Comment (aappling-usgs) [inline js/timeslider.js]: this could be `chart_height * 0.95` instead, if desired

File: js/build_map.js [hunk 3]
```diff
@@ -36,6 +36,6 @@ function create_map() {
 	var state_data = arguments[1];
   
   add_states(map, state_data);
-  add_timeslider(map, years, chart_width, chart_width);
+  add_timeslider(map, years, chart_width, chart_height);
```
Comment (aappling-usgs) [inline js/build_map.js]: ah hah!

File: js/timeslider.js (hunk 2 above)
Comment (lindsaycarr) [reply js/timeslider.js]: oh duh! I will fix in a future PR

**Pull request merged**